trajectorytools/trajectories/concatenate.py

```diff
@@ -58,20 +61,60 @@ def _concatenate_idtrackerai_dicts(traj_dicts):
     """
     traj_dict_cat = traj_dicts[0].copy()
     traj_cat = _concatenate_np(
-        [traj_dict["trajectories"] for traj_dict in traj_dicts]
+        [traj_dict["trajectories"] for traj_dict in traj_dicts], **kwargs
```

Comment:
Why do you send **kwargs to _concatenate_np? _concatenate_np only has one positional argument

Reply:
Hmm indeed, I think I put it there to be able to change the shape without changing the code. I removed the argument from the function's signature, but the kwargs lived on. Should be removed

Reply:
Dealt with here https://github.com/fjhheras/trajectorytools/pull/43/commits/e8e3f81cc97b4ba72502d6fad26ef6dcff63c4ed